screens/ClerkLoginScreen.js

```diff
@@ -99,48 +70,59 @@ export default class ClerkLogin extends React.Component {
             }
           }
         );
+    }).catch(err => {
+      console.error("Error looking up clerk", err);
     });
   }
 
   // From SignUpScreen. Sign in function. It sets the user token in local storage
   // to be the fname + lname and then navigates to homescreen.
   _asyncSignin = async recordId => {
     await AsyncStorage.setItem("clerkId", recordId);
+    await AsyncStorage.setItem("storeId", this.state.storeId);
     this.props.navigation.navigate("CustomerPhoneNumberScreen");
     // this.props.navigation.navigate("ClerkLoginScreen");
   };
 
   // This function will sign the user in if the clerk is found.
   async handleSubmit() {
+    await BASE("Stores")
+      .find(this.state.storeId)
+      .then(storeRecord => {
```

Comment:
This needs a `.catch` to handle possible Airtable API errors

Reply:
Done!